ThingAppraiser/input/Input.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace ThingAppraiser.input
+{
+    public class Input
+    {
+        public string[] ReadLines(string fileName)
+        {
+            List<string> res = new List<string>();
+            using (var reader = new StreamReader(fileName))
+            {
+                string line;
+                while ((line = reader.ReadLine()) != null)
+                {
+                    res.Add(line.Trim(new[] { '\r', '\n', ' ' }));
```

Comment:
You can use trim method without creating array explicit (because this method has [`params`](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params) keyword in its definition):

```C#
res.Add(line.Trim('\r', '\n', ' '));
```